test/mongodb.test.js

```diff
@@ -82,10 +82,19 @@ describe('mongodb connector', function() {
       power: { type: String, index: true, unique: true },
       address: { type: String, required: false, index: { mongodb: { unique: false, sparse: true }}},
       description: { type: String, required: false },
-      geometry: { type: Object, required: false, index: { mongodb: { kind: '2dsphere' }}},
```

Comment:
@viki53 Maybe we can avoid all the above as suggested by @bajtos by creating your own index (and model if necessary) instead of reusing Superhero? Thoughts?